Testing/Unit/PowerShell/Connection/Connect-Tenant.Tests.ps1

```diff
@@ -0,0 +1,235 @@
+Import-Module (Join-Path -Path $PSScriptRoot -ChildPath "../../../../PowerShell/ScubaGear/Modules/Connection/Connection.psm1") -Function 'Connect-Tenant' -Force
+
+InModuleScope Connection {
+    Describe -Tag 'Connection' -Name 'Connect-Tenant' {
```

Comment:
While looking at unit under test, I see it assumes the ProductNames is a set instead of a list.  Since the algorythm in Connect-Tenant assumes a set we should probally write an issue so do a Get-Unique when we Sort-Object in the Orchestrator.

Reply:
Thanks for catching that. We will definitely make sure that the list of the product names are unique and sorted so that there are no errors when handling them as a set.